apps/playlist/apis/playlist/v0alpha1/playlist_spec_gen.go

```diff
@@ -0,0 +1,36 @@
+package v0alpha1
+
+// Defines values for PlaylistItemType.
```

Comment:
@IfSentient does this come from https://github.com/grafana/grafana-app-sdk/blob/main/internal/deepmap/oapi-codegen/pkg/codegen/templates/constants.tmpl#L9 ?  or is that just coincidence that they both say "Defines values for"